packages/react/lib/template.js

```diff
@@ -25,6 +25,7 @@ module.exports = ({
     ${fragments.noscript || ''}
     <script>
       ${Object.entries(globals)
+        .filter(([key]) => key !== '_env')
```

Comment:
This takes away our ability to have environment variables at runtime (https://github.com/xing/hops#environment-variables)